install.sh

```diff
@@ -277,39 +284,66 @@ git sparse-checkout set packages lib bin crew tools install.sh
 git reset --hard origin/"${BRANCH}"
 echo -e "${RESET}"
 
-echo -e "${YELLOW}Updating crew package information...${RESET}\n"
+echoinfo "Updating crew package information...\n"
 # Without setting LD_LIBRARY_PATH, the mandb postinstall fails
 # from not being able to find the gdbm library.
 export LD_LIBRARY_PATH=$(crew const CREW_LIB_PREFIX | sed -e 's:CREW_LIB_PREFIX=::g')
 # Since we just ran git, just update package compatibility information.
 crew update compatible
 
-echo -e "${YELLOW}Installing core Chromebrew packages...${RESET}\n"
+echoinfo "Installing core Chromebrew packages...\n"
 yes | crew install core
 
-echo -e "\n${YELLOW}Running Bootstrap package postinstall scripts...${RESET}\n"
+echoinfo "\nRunning Bootstrap package postinstall scripts...\n"
 crew postinstall $BOOTSTRAP_PACKAGES
 
+echo "                       . .
+                   ..,:;;;::'..
+                 .':lllllllool,.
+                ...cl;..... ,::;'.
+               .'oc...;::::..0KKo.
+               .'od: .:::::, lolc.
+             .'lNMMMO ;ooc.,XMMWx;:;.
+            .dMMMMMMXkMMMMxoMMMMMMMMO.
+            .:O0NMMMMMMMMMM0MMMMMN0Oc.
+              .:xdloddddddoXMMMk:x:....
+              .xMNOKX0OOOOxcodlcXMN0O0XKc.
+              .OMXOKXOOOOOk;ol:OXMK...;N0.
+              'XMKOXXOOOOOk:docOKMW,  .kW;
+             .cMMKOXXOOOOOOOOOOO0MM;  .lMc.
+             .cMM00XKOOOOkkkkkkOOWMl. .cMo.
+             .lMWO0XKOOOkkkkkkkkONMo.  ;Wk.
+             .oMNO0X0OOkkkkkkkkkOXMd..,oW0'
+             .xMNO0X0OOkkkkkkkkkkXMWKXKOx;.
+             .0MXOOOOOOkkkkkkkkkOKM0..
+             'NMWNXXKK000000KKXNNMMX.
+             .;okk0XNWWMMMMWWNKOkdc'.
+                .....'cc:cc:''..."
+echo "  ___  _                               _
+ / (_)| |                             | |
+|     | |     ,_    __   _  _  _    _ | |   ,_    _
+|     |/ \   /  |  /  \_/ |/ |/ |  |/ |/ \_/  |  |/  |  |  |_
+ \___/|   |_/   |_/\__/   |  |  |_/|__/\_/    |_/|__/ \/ \/
```

Comment:
Whoa, that is some fancy ascii art!  It might make sense to add more of a hump on the `h`?

Reply:
yeah what would that look like?

Reply:
I was thinking similar to the `r`.

Reply:
oh okay

Reply:
I think there is an ascii art generator I used ~ 20+ years ago to generate a signature in my email.... There's zero chance of me finding that now, alas..

Reply:
it's okay i used figlet, it worked well enough.

Reply:
That looks better but I'm confused by the extra vertical line between the `C` and `h`.

Reply:
this is a cursive script, look at a cursive h
also because of the `echo '` line, the top line of characters is moved over

Reply:
Fair enough.  I probably don't know how to write in cursive now anyway since it's been so long since I did any actual writing.  Does that make me illiterate?